docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM mrcide/hiv-orderly:latest
 
-RUN install_remote \
-    mrc-ide/threemc
+RUN install_packages --repo="https://mrc-ide.r-universe.dev" \
```

Comment:
do we not need to add CRAN in here too, or does that all work ok?

Reply:
`install_packages` script adds default repos to the list https://github.com/mrc-ide/threemc-orderly-image/blob/main/bin/install_packages#L9